pages/pipelines/dependencies.md.erb

```diff
@@ -90,12 +90,18 @@ Even though the second command step in the above example is after a wait step, t
 
 ## Order of operations
 
-There are three step attributes that can each affect when a step is able to run: `if`/`branches`, `depends_on`, and `concurrency_group`. They are applied to the step in the order they are listed.
+There are three step attributes that can each affect when a step is able to run:
+
+* `if`/`branches`
+* `depends_on`
+* `concurrency_group`
 
 If the step you're dependent on doesn't exist, the build will fail without running the step that is waiting for the dependency.
```

Comment:
@harrietgrace here do you mean if it excluded by a branches condition?

Reply:
Yep, or if you mis-typed the name, or a pipeline upload didn't work... anything that might cause a step to not be there.